### PR TITLE
Feat(SMA-663): Minimal Reproducible Example request

### PR DESCRIPTION
**Pull request by joepegler**
Updated the bug report guideline to more forcibly request a "Minimal Reproducible Example"

### CODE REVIEW

File: packages/account/src/index.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export {
   type FeeQuotesOrDataResponse,
   createPaymaster,
 } from "@biconomy/paymaster";
-export { EthersSigner, convertSigner, type LightSigner } from "@biconomy/common";
+export { EthersSigner, convertSigner, type LightSigner, type SupportedSigner } from "@biconomy/common";
```
Comment (livingrockrises) [inline packages/account/src/index.ts:19]: what is this change for? does it have to be part of this PR purpose?

**Review by livingrockrises — APPROVED** [review]
reviewed

File: packages/account/src/index.ts (hunk 1 above)
Comment (joepegler) [reply packages/account/src/index.ts:19]: No it's not relevant to this PR. I noticed we'll need to export this if the version of viem that we bundle gets dated - users will need to cast their signer. The long term solution for this is to move viem to a peer dep.

**Comment by livingrockrises** [comment]
conflict with lockfile

**Comment by joepegler** [comment]
fixed conflicts

**Review by VGabriel45 — APPROVED** [review]

**Pull request merged**